include/vapor/BarbParams.h

```diff
@@ -42,17 +42,17 @@ class PARAMS_API BarbParams : public RenderParams {
  //! E.g. the grid on which barbs are placed.
  //! \retval vector<long> grid
  const vector<long> GetGrid() const {
-	vector<long> defaultGrid(2, 10);
-	defaultGrid.push_back(1.0);
-	return (GetValueLongVec(_gridTag,defaultGrid));
+     vector<long> dims(3);
+     dims[0] = GetValueLong(_xBarbsCountTag, 10);
```

Comment:
changing the parameter tags will make older session files incompatible, will it not? It would be good to avoid this if we can, but we can discuss.

Reply:
In the sense that certain renderers may lose a part of their saved state but the session files will still be 95% ok. It is possible to achieve similar functionality without making these changes. We can discuss pros/cons at our meeting.